doc/specs/2026-03-29-firestore-to-rdb-migration-design.md

```
@@ -0,0 +1,486 @@
+
+### コスト試算
+
+| DAU | Neon | CF Workers | 月額合計 |
+|---|---|---|---|
+| ~1,000 | Free | Free | **$0** |
+| ~3,000 | Launch $19 | Paid $5 | **$24** |
+| ~10,000 | Scale $69 | Paid $9 | **$78** |
+
+## 3. 全体アーキテクチャ
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)